Lib/test/test_enum.py

```diff
@@ -4487,11 +4493,17 @@ def test_convert(self):
                          [], msg='Names other than CONVERT_TEST_* found.')
 
     def test_convert_uncomparable(self):
-        uncomp = enum.Enum._convert_(
-            'Uncomparable',
-            MODULE,
-            filter=lambda x: x.startswith('UNCOMPARABLE_'),
-        )
+        # We swap a module to some other object with `__dict__`
+        # because otherwise refleak is created.
+        # `_convert_` uses a module side effect that does this. See 30472
+        with support.swap_item(
+            sys.modules, MODULE, _ModuleWrapper(sys.modules[MODULE]),
+        ):
```

Comment:
Do not use Black formatting on enum code, please.   Continuation lines should be eight space indents, and each closing `)`, `]`, and `}` should line up as if it were code and not syntax, with the final line (if it introduces a block) only indented the normal four spaces.  In other words:

    a_long_list_comprehension = [
            x
            for x in spam(eggs, ham)
            if x not in some_previous_group
            ]

and 

    with yada(
            arg1, arg2, something_else=None,
        ):
        # body of code